### ISSUE TITLE
Bug downloading the atomic dataset

### ISSUE THREAD

**Issue by morganlee123** [tardis-sn/tardis#1288]
#### Code Sample, a copy-pastable example if possible

```python
from tardis import run_tardis
from tardis.io.atom_data.util import download_atom_data

# the data is automatically downloaded
download_atom_data('kurucz_cd23_chianti_H_He')
```
![Screen Shot 2020-09-01 at 4 26 52 PM](https://user-images.githubusercontent.com/23300342/91902618-ff7b0a00-ec6f-11ea-877a-a2af1a4d0dee.png)

#### Problem description

Atomic dataset is not downloading. Returning a 404 not found with the githubusercontent link. Cell is completing in python notebook instantly without error

#### Expected Output

The atomic dataset downloads properly to computer

**Comment by epassaro** [comment]
This is not strictly a bug. 

We ran out of LFS quota and we can't access those files for 30 days.

**Comment by epassaro** [comment]
Closed by #1291
